plugins/item_tasks/web_client/templates/itemMenuMod.pug

```diff
@@ -11,5 +11,9 @@ if currentUser && currentUser.get('admin')
     a.g-configure-item-task(role="menuitem")
       i.icon-cube
       | Configure task
+  li(role="presentation")
+    a.g-select-item-task(role="menuitem")
+      i.icon-search
+      | Select task
```

Comment:
This should be a more descriptive string. I don't have a great idea, but maybe something like `Run a task on this item` (but probably shorter than that).